.github/workflows/npm-release-publish.yml

```diff
@@ -25,6 +25,15 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  workflow-info:
+    runs-on: ubuntu-latest
+    steps:
+      - name: job summary
+        run: |
+          echo "### branch: ${GITHUB_REF##*/}" >> $GITHUB_STEP_SUMMARY
```

Comment:
nit: the branchname being in a header doesn't suggest to me it's something I should review:
![image](https://user-images.githubusercontent.com/2684369/204929303-a1e69e80-5ce6-42df-a58a-895db1c5e88f.png)

(Just suggestions, can ignore) Consider moving it inline with the other inputs, adding a comment to anyone looking at the summary why its there, and putting the reviewed parts in code markers to stand out, so the markdown summary would turn out like:

```md
### Job Inputs - Review before approving workflow

This job has been triggered with the following inputs, review these are correct before approving the npm deployment.

- release branch: `release/1.4.1`
- npm-tag: `next`
- release-flavor: `beta`
```

Resulting in:

### Job Inputs - Review before approving workflow

This job has been triggered with the following inputs, review these are correct before approving the npm deployment.

- release branch: `release/1.4.1`
- npm-tag: `next`
- release-flavor: `beta`